src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/PrisonerSearchController.kt

```diff
@@ -16,6 +16,7 @@ import java.time.LocalDate
 class PrisonerSearchController(private val prisonerSearchService: PrisonerSearchService) {
 
   @GetMapping
+  @PreAuthorize(HAS_VIEW_INDUCTIONS)
```

Comment:
Did we not want to use your new `HAS_SEARCH_PRISONS` here? 🤔

Reply:
it didn't work so I've changed it to this for now.